README.markdown

```diff
@@ -95,6 +95,65 @@ end
 
 See `lib/fakefs/spec_helpers.rb` for more info.
 
+**NOTE**: There is a bug currently, that causes FakeFS to fail with the following error:
+```
+TypeError:
+  superclass mismatch for class File
+```
+
+To prevent this, you'll need to require 'pp' *BEFORE* you require 'fakefs/spec_helpers' in your
+spec_helpers file. Like this:
+
+```ruby
+require 'pp'
+require 'fakefs/spec_helpers'
```

Comment:
can you put the require into the spec_helper (or remove what causes the bug) and remove this note ?